frontend/content/faq.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "label": "Kiedy odbędzie się **kolejna** edycja Hack4Krak CTF?",
```

Comment:
<img width="942" height="201" alt="image" src="https://github.com/user-attachments/assets/d2268115-2403-42de-8579-09a31c9a8461" />

Do not use bold in label. If you really need this, please open a new issue